looprpc/server.proto

```diff
@@ -206,9 +206,12 @@ message ServerLoopInResponse {
 }
 
 message ServerLoopInQuoteRequest {
-    /// The swap amount. If zero, a quote for a maximum amt swap will be given.
+    // The swap amount. If zero, a quote for a maximum amt swap will be given.
```

Comment:
We copy this file across from the server (to keep in sync), so I wouldn't advise making aesthetic changes here unless they're done server-side as well. Can be pulled out into a separate PR if we want to clean things up?